gc/base/GCExtensionsBase.hpp

```diff
@@ -763,6 +763,36 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 #endif /* defined(OMR_GC_OBJECT_MAP) */
 
 	MMINLINE bool
+	isScavengerEnabled()
+	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
+		return scavengerEnabled;
+#else
+		return false;
+#endif /* defined(OMR_GC_MODRON_SCAVENGER) */
+	}
+
+	MMINLINE bool
+	isConcurrentMark()
```

Comment:
Can we change this function to isConcurrentMarkEnabled()?  This way the functions all have similar names.